### DISCUSSION TITLE
How to resolve certificate errors in Node mode?

### DISCUSSION THREAD

**Discussion by shanitang** [caoccao/Javet#233] [Q&A] [answered]
I was trying to send https request from Node mode, and had the `UNABLE_TO_GET_ISSUER_CERT_LOCALLY ` error.  What would be the proper way of specifying the CA certificate?

**Comment by caoccao** [comment]
That depends on the http module you use. Basically, you may manually read some CA files via `fs` module for the http module to make the requests or serve the requests. There are plenty of articles online for that.

**Comment by shanitang** [comment]
Is there a preferable way of setting up `NODE_EXTRA_CA_CERTS` environment variable?

**Comment by caoccao** [comment, marked as answer]
> Is there a preferable way of setting up `NODE_EXTRA_CA_CERTS` environment variable?

Just set env variables to the JVM.

**Reply by shanitang** [reply]
Thank you!
